internal/pkg/manifest/lb_web_svc.go

```diff
@@ -104,11 +105,43 @@ func newDefaultLoadBalancedWebService() *LoadBalancedWebService {
 		},
 		LoadBalancedWebServiceConfig: LoadBalancedWebServiceConfig{
 			ImageConfig: ImageWithPortAndHealthcheck{},
-			RoutingRule: RoutingRule{
-				HealthCheck: HealthCheckArgsOrString{
-					HealthCheckPath: aws.String(DefaultHealthCheckPath),
+			RoutingRule: RoutingRuleConfigOrBool{
+				RoutingRuleConfiguration: RoutingRuleConfiguration{
+					HealthCheck: HealthCheckArgsOrString{
+						HealthCheckPath: aws.String(DefaultHealthCheckPath),
+					},
+				},
+			},
+			TaskConfig: TaskConfig{
+				CPU:    aws.Int(256),
+				Memory: aws.Int(512),
+				Count: Count{
+					Value: aws.Int(1),
+					AdvancedCount: AdvancedCount{ // Leave advanced count empty while passing down the type of the workload.
+						workloadType: LoadBalancedWebServiceType,
+					},
+				},
+				ExecuteCommand: ExecuteCommand{
+					Enable: aws.Bool(false),
+				},
+			},
+			Network: NetworkConfig{
+				VPC: vpcConfig{
+					Placement: &PublicSubnetPlacement,
 				},
 			},
+		},
+	}
+}
+
+// newDefaultEmptyLoadBalancedWebService returns an empty LoadBalancedWebService with only the default values set, without any load balancer configured.
+func newDefaultEmptyLoadBalancedWebService() *LoadBalancedWebService {
```

Comment:
Can we try to reduce the duplication between `newDefaultEmptyLoadBalancedWebService` and `newDefaultLoadBalancedWebService`?